resources/application-connector/charts/application-operator/templates/controller.yaml

```diff
@@ -42,4 +42,12 @@ spec:
           requests:
             cpu: 100m
             memory: 20Mi
+        volumeMounts:
+          - mountPath: /etc/certs
```

Comment:
You have added certs to app-broker and what about helm broker? Does helm broker owners know how to handle this change?

Reply:
Yes, they have been informed, but will add their changes in this PR at a later date